validator/specs/lib/validateTilesetSpec.js

```diff
@@ -241,6 +241,82 @@ describe('validateTileset', function() {
             }), done).toResolve();
     });
 
+    it('returns error message when content\'s sphere type boundingVolume is entirely outside tile\'s box type boundingVolume', function(done) {
+        var tileBoundingVolume = {
+            box: [
+                0, 0, 0,
+                7, 0, 0,
+                0, 3, 0,
+                0, 0, 5
+            ]
+        };
+        var contentBoundingVolume = {
+            sphere: [10, 10, 10, 1]
+        };
+        var tileset = createSampleTileset(tileBoundingVolume, contentBoundingVolume);
+        expect(validateTileset(tileset)
+            .then(function(message) {
+                expect(message).toBe('content sphere [' + contentBoundingVolume.sphere + '] is not within tile box [' + tileBoundingVolume.box + ']');
+          }), done).toResolve();
+    });
+
+    it('returns error message when content\'s center is within tile\'s boundingVolume, but content\'s sphere type boundingVolume is not', function(done) {
```

Comment:
This description could be clearer, maybe: `returns error message when content bounding sphere center is within the tile's bounding box but its radius extends beyond`.